src/Stratis.Bitcoin.IntegrationTests/Mempool/MemoryPoolTests.cs

```diff
@@ -350,8 +350,6 @@ public void MempoolSyncTransactions()
                 // generate blocks and wait for the downloader to pickup
                 stratisNodeSync.SetDummyMinerSecret(new BitcoinSecret(new Key(), stratisNodeSync.FullNode.Network));
                 stratisNodeSync.GenerateStratisWithMiner(105); // coinbase maturity = 100
-                // wait for block repo for block sync to work
```

Comment:
don't we need to check if the node is synced?

Reply:
The test failed for me unless I removed that. I tried it repeatedly.

Reply:
A test should never fail if we wait a bit longer for a node to be synced.
There is only one test that we know always fail and it's "A_cut_off_miner_advanced_ahead_of_network_causes_reorg_on_reconnect".

Reply:
The test does fail if it takes the node longer than the helper method's timeout to sync. Why it might be taking longer than that, I do not know.